README.md

```diff
@@ -53,24 +53,33 @@ Help for `alchemical_analysis.py` (obtained with `alchemical_analysis -h`) is:
   -h, --help            show this help message and exit
   -a SOFTWARE, --software=SOFTWARE
                         Package's name the data files come from: Gromacs,
-                        Sire, or AMBER. Default: Gromacs.
+                        Sire, Desmond, or AMBER. Default: Gromacs.
+  -b BREAKS, --break=BREAKS
+                        Extract the portion of the output files and calculate
+                        the data based on that portion. Default: 1.0
   -c, --cfm             The Curve-Fitting-Method-based consistency inspector.
                         Default: False.
   -d DATAFILE_DIRECTORY, --dir=DATAFILE_DIRECTORY
                         Directory in which data files are stored. Default:
                         Current directory.
+  -e, --backward        Using the backward direction for breaking the output
```

Comment:
I don't understand this doc string either. What's "backwardi" and how would one break the output file in teh backward direction?

Reply:
@davidlmobley if using this option, the energy data will be extracted from the output file using the backward direction, yes, it's not break the output file but just extract them. I made a change to this doc string.